catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/QueryResultIterable.java

```diff
@@ -65,8 +68,7 @@ public QueryResultIterable(CatalogFramework catalog, Function<Integer, QueryRequ
      *                 the caller can control iteration based on their own logic
      * @param pageSize How many results should each page hold
      */
-    public QueryResultIterable(CatalogFramework catalog, Function<Integer, QueryRequest> filter,
-            int pageSize) {
+    public QueryResultIterable(CatalogFramework catalog, Filter filter, int pageSize) {
         this.catalog = catalog;
```

Comment:
@mweser, like we discussed, I don't think the page size makes sense here. It only makes sense if we have a separate class returning batched results, in which case the argument should probably be called `batchSize` instead and have to be `>= 1`.

For this class, we probably should just get rid of the `pageSize` argument altogether.